server/auth/routes.js

```diff
@@ -15,7 +16,7 @@ router.post("/login", async (req, res, next) => {
 		if (!email || !password || !user || password !== user.password) {
 			return res.sendStatus(403);
 		}
-
+	
 		const secret = process.env.JWT_SECRET || "your_jwt_secret";
 		const token = jwt.sign({ userId: user.id }, secret);
 		delete user.password;
```

Comment:
Try not to leave unnecessary whitespace changes in a file. If you want to undo changes from a previous commit you can either reset the branch, or use the `git revert` command.